### PR TITLE
This PR completes Mess Section

### PR DESCRIPTION
**Pull request by mtabishk**
Adds following changes:
- Mess Menu Student UI and API Integration
- Mess Menu Manager UI and API Integration
- Mess Complaint and API Integration
- Updated MOM and Tenders UI

https://user-images.githubusercontent.com/61789893/189485271-4a48bc3f-d3de-4dd4-9884-744499e3ca7d.mp4

https://user-images.githubusercontent.com/61789893/189485341-3546bb6a-a9ea-499d-91ff-e017108baf41.mp4

### CODE REVIEW

File: app/.gitignore [hunk 1]
```diff
@@ -43,4 +43,4 @@ app.*.map.json
 # Android Studio will place build artifacts here
 /android/app/debug
 /android/app/profile
-/android/app/release
\ No newline at end of file
+upload-keystore.jks
```
Comment (ashutoshc8101) [inline app/.gitignore]: Revert this change

File: app/android/app/build.gradle [hunk 2]
```diff
@@ -50,12 +50,12 @@ android {
...
+ signingConfigs {
         debug {
-            storeFile file('key.jks')
-            storePassword 'instituteapp'
-            keyAlias 'androiddebugkey'
-            keyPassword 'instituteapp'
+            storeFile file('upload-keystore.jks')
+            storePassword 'password'
+            keyAlias 'uploadkeystore'
+            keyPassword 'password'
```
Comment (ashutoshc8101) [inline app/android/app/build.gradle:58]: Revert these changes.

**Review by ashutoshc8101 — CHANGES_REQUESTED** [review]

**Comment by ashutoshc8101** [comment]
@Jeetu303 Please review the video.

File: app/.gitignore (hunk 1 above)
Comment (mtabishk) [reply app/.gitignore]: Done

File: app/android/app/build.gradle (hunk 2 above)
Comment (mtabishk) [reply app/android/app/build.gradle]: Done

**Comment by mtabishk** [comment]
@ashutoshc8101 Changes done

**Review by ashutoshc8101 — APPROVED** [review]

**Comment by Jeetu303** [comment]
All this looks good in both videos. 
When you add an item or change the time of a meal and update it and go to the student's side, an update should get reflected there also. Show this process in one video... Do not add or change via Django administration... Login manager side, do changes then login as a student... @ashutoshc8101 @mtabishk

**Comment by Jeetu303** [comment]
Mess Complaint and API Integration
Updated MOM and Tenders UI
I cannot see videos for the above ones @mtabishk @ashutoshc8101

**Comment by Jeetu303** [comment]
@mtabishk @ashutoshc8101 do this please

**Comment by ashutoshc8101** [comment]
@mtabishk Flutter analyze is failing.

**Comment by mtabishk** [comment]
@ashutoshc8101 
flutter analyze

Its not failing on my end

**Comment by Jeetu303** [comment]
When the manager log-ins it should show all the functionalities ... But shows Mess.... This page is for mess managers... it should directly go to functionalities. Remove that page(video time 1:02 around)...
If the mess manager can edit timing, then everything is fine. 
@mtabishk Do these two things when done merge pr @ashutoshc8101 
Good job keep it up

**Comment by mtabishk** [comment]
@Jeetu303 @ashutoshc8101 
Made the suggested changes 

https://user-images.githubusercontent.com/61789893/190845143-41d7a3eb-afee-4587-8379-8b7807377148.mp4

**Comment by Jeetu303** [comment]
Quoting the last comment again "**If the mess manager can edit timing, then everything is fine.** 
So read comment carefully ... @ashutoshc8101 @mtabishk if that is happening then go ahead and merge if not then do that first then merge

**Comment by mtabishk** [comment]
@ashutoshc8101 Fixed merge conflicts

**Comment by mtabishk** [comment]
@ashutoshc8101 
Mess Timings video

https://user-images.githubusercontent.com/61789893/190846852-f4fe03a9-8b15-4755-9bd1-e6f11cffb833.mp4

**Pull request merged**